brainstormforce/ultimate-addons-for-gutenberg
# Fix: Responsive Control - Resolved console issue when click on add new button that is on plus button.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/999e91fd-4a41-4ad0-ab8e-0e33a57b6554?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Screenshots

- See here - https://share.getcloudapp.com/GGulE1GW

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Go to the editor, Click on add new block i. e + button in the editor.
- Type any block name. See Screencast mention.
- Check issue is resolved or not.

### Checklist:
